drivers/photo-electronic/adpd410x/adpd410x.h

```diff
@@ -0,0 +1,1244 @@
+#define BITP_INT_STATUS_LEV1_INT_LEV1_D         	3
+#define BITP_INT_STATUS_LEV1_INT_LEV1_E         	4
+#define BITP_INT_STATUS_LEV1_INT_LEV1_F         	5
+#define BITP_INT_STATUS_LEV1_INT_LEV1_G         	6
+#define BITP_INT_STATUS_LEV1_INT_LEV1_H         	7
+#define BITP_INT_STATUS_LEV1_INT_LEV1_I         	8
+#define BITP_INT_STATUS_LEV1_INT_LEV1_J         	9
+#define BITP_INT_STATUS_LEV1_INT_LEV1_K         	10
+#define BITP_INT_STATUS_LEV1_INT_LEV1_L         	11
+#define BITM_INT_STATUS_LEV1_INT_LEV1_A         	0x0001
+#define BITM_INT_STATUS_LEV1_INT_LEV1_B         	0x0002
+#define BITM_INT_STATUS_LEV1_INT_LEV1_C         	0x0004
+#define BITM_INT_STATUS_LEV1_INT_LEV1_D         	0x0008
+#define BITM_INT_STATUS_LEV1_INT_LEV1_E         	0x0010
+#define BITM_INT_STATUS_LEV1_INT_LEV1_F         	0x0020
+#define BITM_INT_STATUS_LEV1_INT_LEV1_G         	0x0040
+#define BITM_INT_STATUS_LEV1_INT_LEV1_H         	0x0080
+#define BITM_INT_STATUS_LEV1_INT_LEV1_I         	0x0100
+#define BITM_INT_STATUS_LEV1_INT_LEV1_J         	0x0200
+#define BITM_INT_STATUS_LEV1_INT_LEV1_K         	0x0400
+#define BITM_INT_STATUS_LEV1_INT_LEV1_L         	0x0800
+
+
+/* ADPD410X_REG_FIFO_TH */
+#define BITP_FIFO_CTL_FIFO_TH                   	0
+#define BITM_FIFO_CTL_FIFO_TH                   	0x03ff
+
+/* ADPD410X_REG_INT_ACLEAR */
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_A       	0
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_B       	1
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_C       	2
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_D       	3
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_E       	4
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_F       	5
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_G       	6
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_H       	7
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_I       	8
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_J       	9
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_K       	10
+#define BITP_INT_ACLEAR_INT_ACLEAR_DATA_L       	11
+#define BITP_INT_ACLEAR_INT_ACLEAR_FIFO         	15
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_A       	0x0001
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_B       	0x0002
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_C       	0x0004
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_D       	0x0008
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_E       	0x0010
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_F       	0x0020
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_G       	0x0040
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_H       	0x0080
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_I       	0x0100
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_J       	0x0200
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_K       	0x0400
+#define BITM_INT_ACLEAR_INT_ACLEAR_DATA_L       	0x0800
+#define BITM_INT_ACLEAR_INT_ACLEAR_FIFO         	0x8000
+
+/* ADPD410X_REG_CHIP_ID */
+#define BITP_CHIP_ID_CHIP_ID                    	0
+#define BITP_CHIP_ID_VERSION                    	8
+#define BITM_CHIP_ID_CHIP_ID                    	0x00ff
+#define BITM_CHIP_ID_VERSION                    	0xff00
```

Comment:
Can BITM_CHIP_ID  and BITM_CHIP_VERSION be used?

Reply:
I was going for the naming scheme: REGISTER_NAME_FIELD_NAME. But in this case they coincide. I will change to avoid confusion.